authentication/oauth2-code/python/sdk/README.md

```diff
@@ -0,0 +1,9 @@
+# Authentication With Python Connector and Authorization Code
+1. Create a virtual environment: `python3 -m venv env`.
+1. Activate the virtual environment: `source env/bin/activate`.
+1. Install the required dependencies: `pip install -r requirements.txt`.
+1. Clone the `../.env-code-template` environment file to `.env`.
+1. Update the `.env` file with the appropriate values to match your system details.
+1. Source the environment file `source .env` and provide the sensitive values. Sensitive values should not be stored in the `source.env` file itself.
```

Comment:
This line is not great (meaning it's also bad for the connector). Consider:
 ```suggestion
1. Source the environment file  by executing `source .env`. 
```
There's no real sensitive information in this file.